beacon-chain/core/state/state.go

```diff
@@ -65,12 +65,18 @@ func GenesisBeaconState(deposits []*pb.Deposit, genesisTime uint64, eth1Data *pb
 		}
 	}
 
-	latestBlockRoots := make([][]byte, params.BeaconConfig().HistoricalRootsLimit)
-	for i := 0; i < len(latestBlockRoots); i++ {
-		latestBlockRoots[i] = zeroHash
+	// Purposely set as HISTORICAL_ROOTS_LIMIT as we determined it is more clear.
```

Comment:
What does this comment mean?

Reply:
I wanted to write a comment to explicitly say where we go off-spec and provide reasoning.